code/ai/aiturret.cpp

```diff
@@ -2130,7 +2130,7 @@ int Num_turrets_fired = 0;
  * Given a turret tp and its parent parent_objnum, fire from the turret at its enemy.
  */
 extern int Nebula_sec_range;
-void ai_fire_from_turret(ship *shipp, ship_subsys *ss)
+void ai_turret_execute_behavior(ship *shipp, ship_subsys *ss)
```

Comment:
Add a comment with the old function name, just in case someone is comparing FSO vs. retail code.

Otherwise this looks good.